assignments/prerequisites/python-programming-advanced.ipynb

```diff
@@ -14,6 +14,8 @@
     "MIT License\n",
```

Comment:
<p>The copyright should be in 3 lines.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-academy/machine-learning/pull/336/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>